FRPs/proposal-antitrust.md

```diff
@@ -0,0 +1,44 @@
+Increasing Competition Using Decentralized Technologies
+
+(1) Overview
+
+The Decentralization Research Center (DRC) is excited to submit the research proposal below. We are a nonprofit organization that supports nonpartisan decentralization research and acts as a hub for critical stakeholders. Our mission is to encourage the equitable ownership, governance, and regulation of emerging technologies that impact our everyday lives.
+
+In our proposed research, we aim to investigate how decentralization, especially as enabled by blockchain technology, can impact competition and offer solutions to monopolistic tendencies in various sectors. Central to this investigation is the role of MEV in fostering competitive environments and their implications for future antitrust policies and regulations. The study is inspired by some of our previous research, the paper we co-authored for the Harvard Kennedy School on equitable ownership and governance, and discussions with MEV builders.
+
+The hypothesis suggests that decentralized structures and technologies, including those underpinned by blockchain, can disrupt traditional power concentrations and foster a more equitable distribution of ownership and governance. The methodology involves a three-phase research process over six months. The first two months will focus on literature review and case study selection, including data collection through interviews and market research. The subsequent two months will be dedicated to writing and peer-reviewing the research paper. In the final phase, the team will draft shorter, accessible research outputs like blog posts and tweet threads, culminating in a presentation at the next DRC summit, which will likely be in DC in June 2024.
+
+The goal is to contribute meaningfully to the discourse on decentralization and its role in promoting competitive, equitable markets, and to ensure that message is ultimately communicated to decision makers who can impact the future of this technology.
+
+(2) Background and Problem Statement
+
+Motivated by the DRC’s mission to promote equitable ownership, governance, and regulation of emerging technologies, this proposal addresses the critical intersection of decentralization and antitrust regulation. Decentralization, a concept rooted in distributing power, ownership, and function away from central points, offers a new paradigm in organizational and technological structures. In the context of rising concerns over the concentration of power in the hands of a few dominant players, particularly in the technology sector, this research seeks to answer:
+
+How does decentralization impact competition in various sectors?
+Can decentralized structures and technologies, particularly those enabled by blockchain technology, provide effective solutions to monopolistic market tendencies?
+What role can MEV play in enabling these solutions?
```

Comment:
see comment above: adjust to "what role can privacy technologies and blockchain networks play..."